From 275db69c078335a5f4545116006ecfe9219cfeff Mon Sep 17 00:00:00 2001
From: Thierry FOURNIER <tfournier@exceliance.fr>
Date: Fri, 6 Dec 2013 10:39:36 +0100
Subject: [PATCH 190/203] BUG/MINOR: map: The map list was declared in the
 map.h file

This bug is harmless and post-dev19, it does not require any backport.
---
 include/types/map.h | 2 +-
 src/map.c           | 2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/include/types/map.h b/include/types/map.h
index 15a0ef8..68f1a36 100644
--- a/include/types/map.h
+++ b/include/types/map.h
@@ -29,7 +29,7 @@
  *
  * "maps" is the list head. This list cotains all the mao file name identifier.
  */
-struct list maps = LIST_HEAD_INIT(maps); /* list of struct map_reference */
+extern struct list maps;
 
 struct map_reference {
 	struct list list;    /* used for listing */
diff --git a/src/map.c b/src/map.c
index 4b1aeb5..81c61eb 100644
--- a/src/map.c
+++ b/src/map.c
@@ -22,6 +22,8 @@
 #include <proto/pattern.h>
 #include <proto/sample.h>
 
+struct list maps = LIST_HEAD_INIT(maps); /* list of struct map_reference */
+
 /* This function return existing map reference or return NULL. */
 static struct map_reference *map_get_reference(const char *reference)
 {
-- 
1.8.1.5

